src/progress.rs

```diff
@@ -50,6 +52,117 @@ impl<'a> Progress<'a> {
         )
         .await
     }
+
+    /// Wait until MeiliSearch processes an update, and get its status.
+    /// 
+    /// interval_ms = The frequency at which the server should be polled. Default = 50ms
+    /// timeout_ms = The maximum time to wait for processing to complete. Default = 5000ms
```

Comment:
```suggestion
    /// `interval` = The frequency at which the server should be polled. Default = 50ms
    /// `timeout` = The maximum time to wait for processing to complete. Default = 5000ms
```
Let's remove the `ms` suffix as we switched to `Duration` without unit.